### PR TITLE
Feature:DashBoardScreen

### PR DESCRIPTION
**Pull request by chinnuGarg**
-I have done the change

### CODE REVIEW

File: app/src/main/java/com/app/shaadi/global/Constants.java [hunk 1]
```diff
@@ -10,4 +10,5 @@
     public static int disLiked = 1;
     //Status not yet changed
     public static int notChanged = 2;
+    //fsffgdfdfd
```
Comment (gargaseem) [inline app/src/main/java/com/app/shaadi/global/Constants.java:13]: @chinnuGarg Please add heh valid comme here

**Review by gargaseem — CHANGES_REQUESTED** [review]
-Please use he good Qua;li code

File: app/src/main/java/com/app/shaadi/global/Constants.java (hunk 1 above)
Comment (chinnuGarg) [reply app/src/main/java/com/app/shaadi/global/Constants.java]: @gargaseem I have resolved it

**Review by gargaseem — APPROVED** [review]
L1 Approved

**Pull request merged**